package.json

```diff
@@ -8,14 +8,15 @@
     "test": "standard && node test/unit",
     "test-integration": "node test/integration",
     "start": "electron . --debug --disable-http-cache",
-    "pack": "npm run build && electron-builder --dir",
-    "dist": "npm run build && electron-builder --publish=onTagOrDraft",
+    "pack": "npm run build:prod && electron-builder --dir",
+    "dist": "npm run build:prod && electron-builder --publish=onTagOrDraft",
     "dev": "npm run build && npm run start",
-    "build": "npm run build-es5 && npm run build-conversations",
+    "build": "npm run build-conversations && npm run build:dev",
```

Comment:
Remove this one. Lets just keep `build:dev` and `build:prod`.

Reply:
(So `.travis.yml` needs to be updated to do `npm run build:prod`)

Reply:
just got rid of `prod`, doesn't really matter too much for us because we aren't shipping this to a client